powersimdata/data_access/data_access.py

```diff
@@ -191,6 +217,7 @@ def __init__(self, root=server_setup.DATA_ROOT_DIR):
         """Constructor"""
         super().__init__(root)
         self._fs = None
+        self._base_fs = None
```

Comment:
Will we need this attribute inn the future?

Reply:
Nope! Thanks for catching that.